From a181507a5143d6367efb1cb31e7f7d2a67bdcd4c Mon Sep 17 00:00:00 2001
Message-Id: <a181507a5143d6367efb1cb31e7f7d2a67bdcd4c.1434670190.git.chang-joon.lee@intel.com>
In-Reply-To: <5c49cb8f884ecf227e44159a5a4931d2c0814d5c.1434670190.git.chang-joon.lee@intel.com>
References: <5c49cb8f884ecf227e44159a5a4931d2c0814d5c.1434670190.git.chang-joon.lee@intel.com>
From: vandita kulkarni <vandita.kulkarni@intel.com>
Date: Tue, 2 Jun 2015 00:47:09 +0530
Subject: [PATCH 6/6] REVERTME [VPG]: drm/i915: Remove intermediate wm/dsparb
 updates

This patch removes intermediate wm update, since we do not have to handle
2 plane cases separately. For display arbiter allocation its always a single
plane or 3 plane now.

REVERTME: This patch is revertme, since it based on the VPG set_display
atomic function which itself is a reverme patch.

Issue: IMINAN-33414
Change-Id: I6cb68d84d95ea27508fd3dbc29525a2cb9f80fa2
Signed-off-by: vandita kulkarni <vandita.kulkarni@intel.com>
---
 drivers/gpu/drm/i915/intel_display.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_display.c b/drivers/gpu/drm/i915/intel_display.c
index 87c2c4a..8c76fdd 100644
--- a/drivers/gpu/drm/i915/intel_display.c
+++ b/drivers/gpu/drm/i915/intel_display.c
@@ -11277,10 +11277,6 @@ static int intel_crtc_set_display(struct drm_crtc *crtc,
 	prev_plane_stat = VLV_PLANE_STATS(dev_priv->prev_pipe_plane_stat, pipe);
 	plane_stat = VLV_PLANE_STATS(dev_priv->pipe_plane_stat, pipe);
 
-	if (IS_CHERRYVIEW(dev))
-		if (hweight32(prev_plane_stat) <=  hweight32(plane_stat))
-			valleyview_update_wm_pm5(intel_crtc);
-
 	/* Check if we need to a vblank, if so wait for vblank */
 	if (intel_dsi_is_enc_on_crtc_cmd_mode(crtc)) {
 		/*
-- 
1.9.1

